monitor.py

```diff
@@ -539,7 +539,10 @@ def get_nightly(build_log_url,build_log_response, job_platform):
         nightly_log_re = re.compile('(Resolved release {}-latest to (\S+))'.format(job_platform), re.MULTILINE|re.DOTALL)
         nightly_log_match = nightly_log_re.search(build_log_response.text)
         if nightly_log_match is None:
-                rc_nightly_log_re = re.compile('(Using explicitly provided pull-spec for release {}-latest \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)
+                if job_platform=="multi":
```

Comment:
Even for RC triggered multi jobs we have to take the nightly details using `Using explicitly provided pull-spec for release {}-latest \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)` so change this condition.